src/main/java/com/shopify/model/fulfillmentOrderApi/ShopifyDeliveryMethod.java

```diff
@@ -0,0 +1,101 @@
+package com.shopify.model.fulfillmentOrderApi;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import org.joda.time.DateTime;
+
+import com.shopify.model.adapters.DateTimeAdapter;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class ShopifyDeliveryMethod {
+
+	public enum MethodType {
+		// No delivery method.
+		NONE("none"),
+		// A delivery to a customer's doorstep.
+		LOCAL("local"),
+		// Items delivered immediately in a retail store.
+		RETAIL("retail"),
```

Comment:
I like having the enum but what if they add a new delivery method. The graqphl api shows this to be a lookup table. Maybe we just make the default enum like 'CUSTOM' or something? Just in case so we don't fail hard on newly created delivery methods.

Reply:
nice catch!
how would that look like in JAVA?
I was thinking about a HashTable, but we'd lose the _enum's elegancy_ right?

Reply:
I think what you have here is good. What i'm suggesting is that
1. We add one more method type to the enum called custom 
2. and then instead of throwing an exception on enum type not found we just use custom. 

But just thinking about it, it will cause problems if we need to write back as custom not a field supported in shopify. I guess we can leave this alone for now . The only other way to handle it would be to store the enum alongside the string and then do above. That way we can write the original value back to shopify if we get an update and need to.